Wall charger test

FGRemastered opened the pull request:
Please look it over once. I haven't tested it, but I can't think of a reason why it shouldn't work.

FGRemastered committed:
Added SOS2 compatibility

- Added SOS 2 compatibility (Android are now immune to decompression)
- Fixed indentation of MedPod Patch (purely cosmetic)

FGRemastered committed:
Merge branch 'DevTest1.4' of https://github.com/RWDevathon/Android-Tiers-Reforged into DevTest1.4

FGRemastered committed:
Implemented Wall Charger

- Added wall charger as child of bedside charger
- Added placeholder graphics (taken from WallVitalsMonitor)
- Rewrote description of bedside charger and adapted wall charger accordingly

FGRemastered commented:
Somehow also got the changes from the prior push lumped into it, my bad!

RWDevathon reviewed (changes requested):
- Androids should not have immunity to decompression. T5's may be, just for the purposes of temporary use until their content expansion - but it should be noted they do NOT inherit from ATR_BaseMechanicalPawn, so this patch is targeting everything BUT T5's. 
- ATR_WallCharger does not require the `<mustBePlacedAdjacentCardinalToAndFacing>` node at all, and should be excluded for simplicity.
- The description for charging ports should not state that there is a high power consumption while idle - it is very small. This is an issue in the devTest branch as well, but I won't change this on my end until after this branch is merged to avoid merge conflicts.

FGRemastered commented:
The first and last one have been fixed, but since I made the wall charger use the other one as a base, wouldn't that cause issues if I don't override it?

FGRemastered committed:
Made the requested changes

- Only T5 are immune to decompression rather than everything else
- Applied the SOS 2 patch to both 1.3 and 1.4
- Fixed misunderstanding regarding how the bedside charger works and changed description accordingly
